src/components/detail-page/ViewerCount.jsx

```
@@ -0,0 +1,12 @@
+// src/components/detail-page/ViewerCount.jsx
+import React from 'react';
+
+const ViewerCount = ({ viewCount }) => {
```

ESLint

[error] 'viewCount' is missing in props validation

(react/prop-types)